pom.xml

```diff
@@ -8,7 +8,7 @@
   </parent>
 
   <artifactId>waggle-dance-parent</artifactId>
-  <version>3.0.1-SNAPSHOT</version>
+  <version>3.1.0-SNAPSHOT</version>
```

Comment:
You'll need to make this change in the pom files for all the sub-modules to. This should do the trick:

`mvn versions:set -DnewVersion=3.1.0-SNAPSHOT -DgenerateBackupPoms=false`

Reply:
That didn't work, but I did it manually :)

Reply:
It should have, but maybe it failed since you'd already done the parent manually but not the others? Anyway, try it next time.